nimare/diagnostics.py

```diff
@@ -288,15 +291,21 @@ def _transform(self, expid, label_map, result):
             target_value_map = "est"
         elif "stat" in result.maps:
             target_value_map = "stat"
+        elif "z_desc-consistency" in result.maps:
```

Comment:
I think the specificity map should be the preference (i.e., `z_desc-specificity`) since the result is what is compared to the base rate of activations across the reference dataset.